### ISSUE TITLE
Possibly add warning when loading Trunk builds?

### ISSUE THREAD

**Issue by lewri** [CorsixTH/CorsixTH#2052]
E,g,

```lua
  print("")
  print("")
  print("---------------------------------------------------------------")
  print("")
  print("Welcome to CorsixTH " .. self:getVersion() .. "!")
  if self:getVersion() == "Trunk" then
    print("Warning: You are using a development build of CorsixTH.")
    print("It is not recommended for general use.")
  end
  print("")
  print("---------------------------------------------------------------")
  print("")
  print("This window will display useful information if an error occurs.")
  print("")
  print("---------------------------------------------------------------")
  print("")
```

**Comment by Alberth289346** [comment]
My first reaction was that you're over-estimating the exhaustive testing that we do with an official release, ie in my view there isn't that much difference between a development version and a release, other than we don't give it a good looking number and we don't patch it.
As such, I don't see much use for this warning.

If you do want the warning, the message in the proposal is quite useless. Saying what's not the intended use doesn't give directions for proper usage. Eg "a screw driver is not for sawing wood." is very true, but it neither points towards how you should use a screw driver nor how you would saw wood. I would still be lost how to perform either goal.

I would reword it to "It should be used for testing new features.", or something in that direction.
